src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,262 @@
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(MockitoExtension.class)
+class MenuServiceTest {
+    private static final UUID DEFAULT_MENU_ID = UUID.randomUUID();
+    private static final String DEFAULT_NAME = "메뉴";
+    private static final BigDecimal DEFAULT_PRICE = BigDecimal.TEN;
+    private static final UUID DEFAULT_MENU_GROUP_ID = UUID.randomUUID();
+
+    @Mock
+    private MenuRepository menuRepository;
+    @Mock
+    private MenuGroupRepository menuGroupRepository;
+    @Mock
+    private ProductRepository productRepository;
+    @Mock
+    private PurgomalumClient purgomalumClient;
+
+    @InjectMocks
+    private MenuService menuService;
+
+    private static List<MenuProduct> defaultMenuProducts() {
+        MenuProduct defaultMenuProduct = new MenuProduct();
+        Product product = defaultProduct();
+        defaultMenuProduct.setProductId(product.getId());
+        defaultMenuProduct.setQuantity(1);
+        defaultMenuProduct.setProduct(product);
+
+        return List.of(defaultMenuProduct);
+    }
+
+    static Menu defaultMenu() {
+        return createMenu(DEFAULT_MENU_ID, DEFAULT_NAME, DEFAULT_PRICE, DEFAULT_MENU_GROUP_ID, defaultMenuProducts());
+    }
+
+    private static Menu createMenu(final UUID ID, final String name, final BigDecimal price, final UUID menuGroupId, final List<MenuProduct> menuProducts) {
+        Menu menu = new Menu();
+
+        menu.setId(ID);
+        menu.setName(name);
+        menu.setPrice(price);
+        menu.setMenuGroupId(menuGroupId);
+        menu.setMenuProducts(menuProducts);
+        menu.setDisplayed(true);
+
+        return menu;
+    }
+
+    private static MenuGroup defaultMenuGroup() {
+        MenuGroup menuGroup = new MenuGroup();
+
+        menuGroup.setId(UUID.randomUUID());
+        menuGroup.setName("테스트 메뉴");
+
+        return menuGroup;
+    }
+
+    private static Product defaultProduct() {
+        Product product = new Product();
+
+        product.setId(UUID.randomUUID());
+        product.setPrice(BigDecimal.TEN);
+        product.setName("테스트 상품");
+
+        return product;
+    }
+
+    @DisplayName("메뉴를 생성 할 수 있다.")
+    @Test
+    void create_menu() {
+        final Menu defaultMenu = defaultMenu();
+
+        given(menuGroupRepository.findById(Mockito.any(UUID.class)))
+                .willReturn(Optional.of(defaultMenuGroup()));
+        given(productRepository.findAllByIdIn(Mockito.any()))
+                .willReturn(List.of(defaultProduct()));
+        given(productRepository.findById(Mockito.any(UUID.class)))
+                .willReturn(Optional.of(defaultProduct()));
+        given(purgomalumClient.containsProfanity(Mockito.any(String.class)))
+                .willReturn(false);
+        given(menuRepository.save(Mockito.any(Menu.class)))
+                .willReturn(defaultMenu);
+
+        final Menu result = menuService.create(defaultMenu);
+        assertThat(result).isNotNull();
+    }
+
+    @DisplayName("가격이 필수이다")
```

Comment:
```suggestion
    @DisplayName("가격이 null 일 결우 Exception을 던진다.")
```
개인적으론 위와 같은 형식으로 테스트의 의도를 나타내곤 합니다 😄